src/constants/tabs.ts

```diff
@@ -32,3 +32,15 @@ export const getCreatorPageTabs = (slug: string): Tab[] => [
     disabled: true,
   },
 ]
+
+export const getComicPageTabs = (slug: string): Tab[] => [
```

Comment:
minor: I don't know why, but I would call this `comicPageTabs` for some reason even it's a function actually.

Reply:
Renamed it.